plugins/airbrake/src/components/AirbrakeWidget/AirbrakeWidget.test.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { ThemeProvider } from '@material-ui/core';
+import { lightTheme } from '@backstage/theme';
+import { AirbrakeWidget } from './AirbrakeWidget';
+import exampleData from './ExampleData';
+
+describe('AirbrakeWidget', () => {
+  it('should render all messages sent from a payload', async () => {
```

Comment:
I would get rid of `should` but that's not a big deal if thats what everyone else is following in community. Also can you replace messages with error messages to be more clear? I know all messages are error messages in airbrake but that just give me more clear idea of what the test is doing.

Reply:
We use `should` a lot so you'll see it all over the codebase, but agree that it's fine to remove. Best reword it to make it read well still though, e.g. `it('renders all ...`

Reply:
Done, thanks for the tip!